autorest/azure/example/main.go

```diff
@@ -18,7 +18,9 @@ import (
 
 const resourceGroupURLTemplate = "https://management.azure.com/subscriptions/{subscription-id}/resourcegroups"
 const apiVersion = "2015-01-01"
-const xplatClientID = "04b07795-8ddb-461a-bbee-02f9e1bf7b46"
+
+const nativeAppClientID = "a87032a7-203c-4bf7-913c-44c50d23409a"
+const resource = azure.AzureresourceManagerScope
```

Comment:
Yeah, I did `%s/Resource/resource/g` to fix a lint error and introduced a bunch of silliness. Travis is green again.....